src/panel_gwalker/_gwalker.js

```diff
@@ -52,7 +52,7 @@ function transformSpec(spec, fields) {
 export function render({ model }) {
   // Model state
   const [appearance] = model.useState('appearance')
-  const [themeKey] = model.useState('theme_key')
+  const [themeKey] = model.useState('theme')
```

Comment:
You changed `theme_key` back to `theme`. `theme_key` is aligned with pygwalker. I also changed to `kernel_computation` to align with pygwalker even I liked `server_computation` more. The only one I've not aligned is `fields` - its called `field_specs` in Pygwalker.

Do we align with Pygwalker or use names we believe are better @philippjfr ?

Reply:
Does it work? I did not see you revert this on the `GraphicWalker`?